src/main/java/aeneas/models/PuzzleLevel.java

```diff
@@ -11,10 +11,12 @@
   
   public PuzzleLevel(Bullpen bullpen) {
     super(bullpen);
+    board = new PuzzleBoard();
```

Comment:
Yeah, the `PuzzleLevel` probably shouldn't be constructing it's own board, unless we are entirely sure that we won't ever want to pass in a different `PuzzleBoard`.